src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -202,7 +202,7 @@ static void wrapped_rr_closure(grpc_exec_ctx* exec_ctx, void* arg,
         gpr_log(GPR_ERROR,
                 "No LB token for connected subchannel pick %p (from RR "
```

Comment:
Should we include the address of the grpclb policy in this log message?

Reply:
`glb_policy` isn't available here. We could add it to `wc_args`...